osm2gtfs/core/osm_connector.py

```diff
@@ -576,6 +563,7 @@ def _query_routes(self):
         query_str = """(
             /* Obtain route variants based on tags and bounding box */
             relation%s(%s)->.routes;
+            //relation(10186002)->.routes;
 
```

Comment:
Seems like a debug line of code. Please remove.

Reply:
indeed, my bad